src/EntityComponentManager.cc

```diff
@@ -766,9 +856,12 @@ void EntityComponentManager::AddEntityToMessage(msgs::SerializedState &_msg,
     compBase->Serialize(ostr);
 
     compMsg->set_component(ostr.str());
-
-    // TODO(anyone) Set component being removed once we have a way to queue it
   }
+
+  // Add a component to the message and set it to be removed if the component
+  // exists in the removedComponents map.
+  // TODO(anyone) Set component being removed once we have a way to queue it
```

Comment:
I think these TODOs can be removed now, right? We're not "queuing" the removal, but for the purposes of this function, keeping track of what components were removed should be all that's needed.

Reply:
Should I move them to where they originally were?
I didn't remove them because I though maybe some kind of queuing should need to be done in the future.

Reply:
I'd just remove these lines. The queuing is still needed, but not by this function

Reply:
b913c32